examples/opencl/fractal.cpp

```diff
@@ -138,6 +138,7 @@ int main(void) {
 
         // destroy GL-CPU Interop buffer
         releaseGLBuffer(handle);
+	context = cl::Context();
```

Comment:
requiring something like this at the end of program indicates most likely some bug in OpenCL implementation I would think.